app/assets/v2/js/pages/increase_bounty.js

```diff
@@ -0,0 +1,160 @@
+load_tokens();
+var setUsdAmount= function (event) {
+    var amount  = $('input[name=amount]').val();
+    var denomination  = $('#token option:selected').text();
+    var estimate = getUSDEstimate(amount, denomination, function(estimate){
+        $('#usd_amount').html(estimate);
+    });
+};
+
+// Wait until page is loaded, then run the function
+$(document).ready(function(){
+    $("input[name=amount]").keyup(setUsdAmount);
+    $("input[name=amount]").blur(setUsdAmount);
+    $("select[name=deonomination]").change(setUsdAmount);
+
+    $('input[name=amount]').focus();
+
+    var denomSelect = $('select[name=deonomination]');
+    localStorage['tokenAddress'] = denomSelect.data('tokenAddress');
+    localStorage['amount'] = 0;
+    denomSelect.select2();
+    $('.js-select2').each(function() {
+        $(this).select2();
+    });
+
+    //submit bounty button click
+    $('#submitBounty').click(function(e){
+        mixpanel.track("Increase Bounty Clicked (funder)", {});
+
+        //setup
+        e.preventDefault();
+        loading_button($(this));
+
+        var issueURL = $('input[name=issueURL]').val();
+        var amount = $('input[name=amount]').val();
+        var tokenAddress = $('select[name=deonomination]').val();
+        var token = tokenAddressToDetails(tokenAddress);
+        var decimals = token['decimals'];
+        var tokenName = token['name'];
+        var decimalDivisor = Math.pow( 10, decimals );
+
+        //validation
+        var isError = false;
+
+        if($('#terms:checked').length == 0){
+            _alert({ message: "Please accept the terms of service." });
+            isError = true;
+        } else {
+            localStorage['acceptTOS'] = true;
+        }
+        var is_issueURL_invalid = issueURL == ''
+            || issueURL.indexOf('http') != 0
+            || issueURL.indexOf('github') == -1
+            || issueURL.indexOf('javascript:') != -1
+        ;
+        if(is_issueURL_invalid){
+            _alert({ message: "Please enter a valid github issue URL." });
+            isError = true;
+        }
+        if(amount == ''){
```

Comment:
maybe worth it to add an error IFF someone tries to do this thats not the original funder

Reply:
Will do.

Reply:
I'm doing this [here](https://github.com/gitcoinco/web/pull/626/files/ed65016827acce33c154bccb7e81996e19878c7c#diff-6bae2171a673cdcc00b148c8898c9ea9R120), do you want me to move it?